RawSpeed/DngDecoder.cpp

```diff
@@ -503,6 +504,37 @@ void DngDecoder::decodeMetaDataInternal(CameraMetaData *meta) {
   if (mRootIFD->hasEntryRecursive(ISOSPEEDRATINGS))
     mRaw->metadata.isoSpeed = mRootIFD->getEntryRecursive(ISOSPEEDRATINGS)->getInt();
 
+  // Set the make and model
+  if (!(mRootIFD->hasEntryRecursive(MAKE) && mRootIFD->hasEntryRecursive(MODEL))) {
+    if (mRootIFD->hasEntryRecursive(UNIQUECAMERAMODEL)) {
```

Comment:
Do you have a test case for this?

Maybe it would make for sense to search the metadata for a match and use the make/model from that if any is found?